### PR TITLE
Feat: Added plugins on cloud-image-builder config

### PR DESCRIPTION
**Pull request by jparrill**
- Allowed automerge by squash method
- Added some plugins in order to start testing soon:
  - config-updater
  - skip
  - override
  - trigger

### CODE REVIEW

File: github/ci/prow/files/plugins.yaml [hunk 1]
```diff
@@ -66,6 +66,10 @@ plugins:
   - approve
   - lgtm
   - owners-label
+  - config-updater
```
Comment (rmohr) [inline github/ci/prow/files/plugins.yaml]: The config-update is enabled for the project-infra repo. It contains all configurations. You don't have to add this.

File: github/ci/prow/files/plugins.yaml [hunk 2]
```diff
@@ -66,6 +66,10 @@ plugins:
...
+  - skip
```
Comment (rmohr) [inline github/ci/prow/files/plugins.yaml]: I think it makes sense to enable `skip` and `override` for the whole org (the `kubevirt:` section directly below `plugins:`.

Comment (jparrill) [reply github/ci/prow/files/plugins.yaml]: Make sense, yes. Almost all will end needing it. Adding them to the org section

**Comment by rmohr** [comment]
@jparrill the validator wants that we remove `override` and `skip` for all subrepositories if we enable it for the whole org.

**Comment by jparrill** [comment]
> @jparrill the validator wants that we remove `override` and `skip` for all subrepositories if we enable it for the whole org.

**Pull request merged**